api/src/convert/account_key.rs

```diff
@@ -7,24 +7,13 @@ use mc_account_keys::AccountKey;
 
 impl From<&AccountKey> for external::AccountKey {
     fn from(src: &AccountKey) -> Self {
-        let mut dst = external::AccountKey::new();
-
-        dst.set_view_private_key(external::RistrettoPrivate::from(src.view_private_key()));
```

Comment:
For many types I dropped the explicit type from and used `into()` or `try_into()`